### PR TITLE
Reviewing the code

### PR DESCRIPTION
**Pull request by dhiemaz**
Hi Chris,

I have made a PR for your code, here are my updates :
1. Fixing typo in the code comment 
2. Removing unnecessary code.
3. adding contributing guideline in readme

### CODE REVIEW

**Comment by krak3n** [comment]
Thank you for the PR, I've been on holiday for the last couple of weeks so will take a look at this shortly 👍

File: README.md [hunk 1]
```diff
@@ -26,6 +26,18 @@ documentation.
...
+Please refer to contribution guidelines for submitting patches and additions. In general, we follow the "fork-and-pull" Git workflow.
+
+ 1. **Fork** the repo on GitHub
+ 2. **Clone** the project to your own machine
+ 3. **Commit** changes to your own branch
+ 4. **Push** your work back up to your fork
+ 5. Submit a **Pull request** so that we can review your changes
+
+NOTE: Be sure to merge the latest from "upstream" before making a pull request!
+
```
Comment (krak3n) [inline README.md:40]: nice 👍

File: example/setup.go [hunk 2]
```diff
@@ -140,6 +140,4 @@ func Localstack(s *ocsns.SNS, q *ocsqs.SQS) error {
 	}
 
 	log.Println("Localstack: Created Subscription:", *sub.SubscriptionArn)
-
-	return nil
```
Comment (krak3n) [inline example/setup.go:144]: I think actually this should return the `err` from `s.Subscribe` and kill the `if err != nil` check

File: ocsns/sns.go [hunk 3]
```diff
@@ -12,7 +12,7 @@ import (
 	"go.opencensus.io/trace"
 )
 
-// An Option function customises a clients configuration
+// An Option function customizes a clients configuration
```
Comment (krak3n) [inline ocsns/sns.go:15]: I'm British so this hurts 😛

**Review by krak3n — CHANGES_REQUESTED** [review]
Thank you for the review and fixing my spelling errors, always was a weak point for me at school 😛 I think the error handling code could be a little cleaner in the example `LocalStack` setup function but otherwise, I'm very happy to accept this PR. If you could make that change I'll get this merged 😄

File: README.md (hunk 1 above)
Comment (dhiemaz) [reply README.md:40]: thank you..

File: ocsns/sns.go (hunk 3 above)
Comment (dhiemaz) [reply ocsns/sns.go:15]: Lol.. So it is better the previous ya @krak3n ?

File: example/setup.go (hunk 2 above)
Comment (dhiemaz) [reply example/setup.go:144]: Yes, you're right.. let me update the code

**Review by dhiemaz — COMMENTED** [review]
@krak3n, I have updated this, adding return err

File: ocsns/sns.go (hunk 3 above)
Comment (krak3n) [reply ocsns/sns.go:15]: nah, I have come to accept American English is the most common form in programming so it's cool :+1:

**Review by krak3n — APPROVED** [review]
Thank you 👍

**Pull request merged**